src/proto-root/proto.attribute.list.js

```diff
@@ -18,14 +20,17 @@ module.exports = {
       const attrValue = attribute.getValue();
       const attrType = attribute.getContentType();
       const processedAnchors = AnchorProcessor.process(attribute.anchors);
-      const convertedValueByType = this.convertValueBasedOnContentType(attrValue, attrType);
+      const convertedValueByType = AttributeConverter
+        .convertValueBasedOnContentType(attrValue, attrType);
       const attrNameInCamelCase = this.toCamelCase(attrName);
 
-      attrList.push({ [attrNameInCamelCase]: convertedValueByType });
-
-      if (attrName === 'selfie') {
-        const imageUriValue = this.imageUriBasedOnContentType(attrValue, attrType);
-        attrList.push({ base64SelfieUri: imageUriValue });
+      // Handle selfies for backwards compatibility.
```

Comment:
I was originally thinking we would not update the selfie attribute to use the new image type yet, only in the major release. But if we have adequate tests to cover making sure this is backwards compatible, then it's fine to do it this way.

Reply:
I've left the API unchanged, but using the new Image object to get the existing values:
- `getBase64SelfieUri()` returns the same value as before (base64 string)
- `selfie` property continues to return the original value (not the new Image object)
-  Existing tests pass